src/backend/profile/profile.service.ts

```diff
@@ -100,3 +101,112 @@ export async function createUserProfile(profileData: CreateProfilePayload) {
   })
   return createdUser
 }
+
+async function updateUserCity(userId: string, newCity: City): Promise<void> {
+  if (!newCity) return
```

Comment:
**nitpick**: We should not do such if without throwing any error. However we should remove this if and always allow user to update city